Wikipedia/Code/WMFWelcomePageViewController.swift

```diff
@@ -12,10 +12,30 @@ public protocol WMFWelcomeNavigationDelegate: class{
     func showNextWelcomePage(_ sender: AnyObject)
 }
 
-class WMFWelcomePageViewController: UIPageViewController, UIPageViewControllerDataSource, UIPageViewControllerDelegate, WMFWelcomeNavigationDelegate {
+class WMFWelcomePageViewController: UIPageViewController, UIPageViewControllerDataSource, UIPageViewControllerDelegate, WMFWelcomeNavigationDelegate, Themeable {
 
     private var theme = Theme.standard
     
+    func apply(theme: Theme) {
+        self.theme = theme
+        guard viewIfLoaded != nil else {
+            return
+        }
+        nextButton.setTitleColor(theme.colors.link, for: .normal)
+        nextButton.setTitleColor(theme.colors.disabledText, for: .disabled)
+        nextButton.setTitleColor(theme.colors.link, for: .highlighted)
+        pageControl?.pageIndicatorTintColor = theme.colors.pageIndicator
+        pageControl?.currentPageIndicatorTintColor = theme.colors.pageIndicatorCurrent
+        skipButton.setTitleColor(theme.colors.unselected, for: .normal)
```

Comment:
Accessing `pageControl` (walking through the subviews to get the first subview that is of type `UIPageControl`) here (view loaded but not appeared) seems to be affecting the Skip button (not getting touch events), still trying to figure out why

Reply:
pushed a fix https://github.com/wikimedia/wikipedia-ios/pull/3227/commits/09922424256acb8c9ff0811b2ca75bc9a2dc25f5